front/components/assistant/conversation/MessageItem.tsx

```diff
@@ -135,14 +139,15 @@ export const MessageItem = React.forwardRef<HTMLDivElement, MessageItemProps>(
           ref={ref}
           className={classNames(
             "mx-auto min-w-60",
-            "pt-6 md:pt-10",
+            userMentionsEnabled ? "mb-4" : "pt-6 md:pt-10",
```

Comment:
I don't think you can remove `pt-6` because it would look like this:

<img width="1730" height="232" alt="Image" src="https://github.com/user-attachments/assets/d8211a85-1205-4923-a8ac-bd55e3f3198e" />

we probably want to add spacing under the date bar?